README.md

```diff
@@ -62,13 +62,28 @@ To test Helm Charts locally you will need to follow the next steps:
 13. Confirm it's gone: `helm list && kubectl get pods`
 14. Shutdown minikube: `minikube stop`
 
-To run unit tests:
+### Running Unit Tests
+To unit test the helm charts tou can follow the next steps
 
 1. Install the unittest plugin for Helm: https://github.com/quintush/helm-unittest
 2. Run the tests for each individual chart:
    * `cd charts/nexus-iq; helm unittest -3 -t junit -o test-output.xml .`
    * `cd charts/nexus-repository-manager; helm unittest -3 -t junit -o test-output.xml .`
 
+### Running Integration Tests
+You can run the integration tests for the helm charts by runnig the next commands. 
+
+Before running the integrations tests:
+* Install docker, helm, kubectl, and [minikube](https://minikube.sigs.k8s.io/docs/start/), if you don't already have it on your local workstation.
+
+Running integration tests for Nexus IQ:
+1. From source code: `helm install iq {path/to/your/helm3-charts}/charts/nexus-iq --wait`
```

Comment:
sometimes when writing this sort of doc, i'll provide paths relative to the root of the project (where this README lives) to keep the paths shorter. I figure you'll recognize the surroundings, and it leaves these samples completely cuttable-and-pastable. Of course, maybe your extra context alleviates confusion that I left in place?

Reply:
Done